app/Http/Controllers/PathController.php

```diff
@@ -55,6 +57,7 @@ public function update(UpdatePath $request, Path $path): JsonResponse
      */
     public function destroy(Path $path): JsonResponse
     {
+        $this->authorize('manage');
```

Comment:
It's one thing copypasted one million times. Maybe you could add this to some middleware? Or even class constructor?